src/Microsoft.Azure.Functions.PowerShellWorker.csproj

```diff
@@ -3,6 +3,7 @@ Copyright (c) Microsoft. All rights reserved.
 Licensed under the MIT license. See LICENSE file in the project root for full license information.
 -->
 <Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Microsoft.Azure.Functions.PowerShellWorker.Common.props))\Microsoft.Azure.Functions.PowerShellWorker.Common.props" />
```

Comment:
Same here, why not `<Import Project="..\Microsoft.Azure.Functions.PowerShellWorker.Common.props" />`?

Reply:
Fixed